jcl/jpp_configuration.xml

```diff
@@ -308,7 +196,7 @@
 		  label="PANAMA"
 		  outputpath="PANAMA/src"
 		  flags="Panama"
-		  dependencies="SIDECAR19-SE-B136"
+		  dependencies="JAVA11"
```

Comment:
JAVA12 would be more appropriate

Reply:
I you make this change then this can be merged.

Reply:
Done.  Thanks, Tobi